src/components/EditCoursePage/index.jsx

```diff
@@ -53,6 +53,9 @@ class EditCoursePage extends React.Component {
       // If a course run triggered the submission, mark it as not a draft
       const submittedRun = courseRuns.find(run => run.key === targetRun.key);
       submittedRun.draft = false;
+      // If we are updating a published course run, we need to also update the course.
+      // We want to use the same indicator of draft = false for consistency.
+      courseData.draft = !(submittedRun.status === PUBLISHED);
```

Comment:
This would read cleaner to me as `courseData.draft = submittedRun.status !== PUBLISHED`

Or honestly, most readable as:
```
if submittedRun.status === PUBLISHED:
    courseData.draft = False
```

Just because setting False to *do* a thing feels weird. And this makes the *doing* a little more explicit.